apps/client/src/context.js

```diff
@@ -1,3 +1,31 @@
-import React from "react";
-export const defaultValues = { topic: null, address: null, checker: null };
-export default React.createContext();
+import React, { useReducer, useEffect, createContext } from "react";
```

Comment:
Maybe context.js is a good candidate to convert to context.ts?

Reply:
Let's decide this after the merge with autofill. We might need to refactor some more.

Reply:
Good idea, I think it's better to do this after out merge with `autofill`